src/logic/operate.js

```diff
@@ -0,0 +1,35 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const a = new Big(numberOne);
+  const b = new Big(numberTwo);
+  const negativeOne = new Big(-1);
+  const hundred = new Big(100);
+  let total;
+
+  switch (operation) {
+    case '+':
+      total = a.plus(b);
+      break;
+    case '-':
+      total = a.minus(b);
+      break;
+    case 'X':
+      total = a.times(b);
+      break;
+    case '÷':
+      total = a.div(b);
+      break;
```

Comment:
dont forget to handle DIVISION BY ZERO on next milestone, else it will generate an error...next TSE will watch this